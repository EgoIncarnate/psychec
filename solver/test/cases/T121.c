
void f() {
    struct T1 v1;
    x1 = v1;
    g1(&v1);

    struct T2* v2;
    x2 = v2;
    g2(v2);
}
