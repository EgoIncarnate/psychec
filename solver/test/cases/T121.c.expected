typedef  struct T2 { int dummy;} T2 ;
typedef  struct T1 { int dummy;} T1 ;
uintptr_t g1 (struct T1*) ;
uintptr_t g2 (struct T2*) ;
struct T1 x1 ;
struct T2* x2 ;
